app/assets/javascripts/discourse/app/components/sidebar/section.hbs

```diff
@@ -1,6 +1,6 @@
 {{#if this.displaySection}}
   <div
-    class={{concat "sidebar-section-wrapper sidebar-section" @class}}
+    class={{concat-class "sidebar-section-wrapper sidebar-section" @class}}
```

Comment:
Bug introduced https://github.com/discourse/discourse/commit/2d46824a87f#diff-950711b37c820496edd808ef3620341e639eab20adb1179ee69d56ce465e63d2R3

concat-class was used before, which adds whitespace. Otherwise we ended with `sidebar-sectiondisabled` class